### PR TITLE
Move LED config settings to the common project

### PR DESCRIPTION
**Pull request by eloots**
This is where these settings should be.

FYI: Tested on Pi cluster for exercises 4 through 13 (where this change applies [at this moment])

### CODE REVIEW

File: common/src/main/resources/application.conf [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  cluster-node-colors {
+    cluster-node-up-color          = green
+    cluster-node-weakly-up-color   = green
+    cluster-node-down-color        = red
+    cluster-node-left-color        = cyan
+    cluster-node-exited-color      = magenta
+    cluster-node-unreachable-color = whiteLow
+    cluster-node-joined-color      = orange
+  }
+}
```
Comment (ironfish) [inline common/src/main/resources/application.conf]: Needs CRLF

File: exercise_004_cluster_base/src/main/resources/application.conf [hunk 2]
```diff
@@ -32,21 +32,4 @@ akka {
...
-    cluster-node-up-color          = green
-    cluster-node-weakly-up-color   = darkgreen
-    cluster-node-down-color        = red
-    cluster-node-left-color        = cyan
-    cluster-node-exited-color      = magenta
-    cluster-node-unreachable-color = whiteLow
-    cluster-node-joined-color      = orange
-  }
-}
+}
```
Comment (ironfish) [inline exercise_004_cluster_base/src/main/resources/application.conf]: Needs CRLF

File: exercise_005_cluster_weakly_up/src/main/resources/application.conf [hunk 3]
```diff
@@ -32,22 +32,4 @@ akka {
...
-    cluster-node-up-color          = green
-    cluster-node-weakly-up-color   = green
-    cluster-node-down-color        = red
-    cluster-node-left-color        = cyan
-    cluster-node-exited-color      = magenta
-    cluster-node-unreachable-color = white
-    cluster-node-joined-color      = orange
-  }
-}
+}
```
Comment (ironfish) [inline exercise_005_cluster_weakly_up/src/main/resources/application.conf]: Needs CRLF

File: exercise_008_cluster_weakly_up_disabled/src/main/resources/application.conf [hunk 4]
```diff
@@ -34,22 +34,4 @@ akka {
...
-    cluster-node-up-color          = green
-    cluster-node-weakly-up-color   = darkgreen
-    cluster-node-down-color        = red
-    cluster-node-left-color        = cyan
-    cluster-node-exited-color      = magenta
-    cluster-node-unreachable-color = whiteLow
-    cluster-node-joined-color      = orange
-  }
-}
+}
```
Comment (ironfish) [inline exercise_008_cluster_weakly_up_disabled/src/main/resources/application.conf]: Needs CRLF

File: exercise_009_split_brain_resolver_keep_majority/src/main/resources/application.conf [hunk 5]
```diff
@@ -54,22 +54,4 @@ akka {
...
-    cluster-node-up-color          = green
-    cluster-node-weakly-up-color   = darkgreen
-    cluster-node-down-color        = red
-    cluster-node-left-color        = cyan
-    cluster-node-exited-color      = magenta
-    cluster-node-unreachable-color = whiteLow
-    cluster-node-joined-color      = orange
-  }
-}
+}
```
Comment (ironfish) [inline exercise_009_split_brain_resolver_keep_majority/src/main/resources/application.conf]: Needs CRLF

File: exercise_010_split_brain_resolver_static_quorum/src/main/resources/application.conf [hunk 6]
```diff
@@ -57,22 +57,4 @@ akka {
...
-    cluster-node-up-color          = green
-    cluster-node-weakly-up-color   = darkgreen
-    cluster-node-down-color        = red
-    cluster-node-left-color        = cyan
-    cluster-node-exited-color      = magenta
-    cluster-node-unreachable-color = whiteLow
-    cluster-node-joined-color      = orange
-  }
-}
+}
```
Comment (ironfish) [inline exercise_010_split_brain_resolver_static_quorum/src/main/resources/application.conf]: Needs CRLF

File: exercise_011_split_brain_resolver_keep_referee/src/main/resources/application.conf [hunk 7]
```diff
@@ -58,22 +58,4 @@ akka {
...
-    cluster-node-up-color          = green
-    cluster-node-weakly-up-color   = darkgreen
-    cluster-node-down-color        = red
-    cluster-node-left-color        = cyan
-    cluster-node-exited-color      = magenta
-    cluster-node-unreachable-color = whiteLow
-    cluster-node-joined-color      = orange
-  }
-}
+}
```
Comment (ironfish) [inline exercise_011_split_brain_resolver_keep_referee/src/main/resources/application.conf]: Needs CRLF

File: exercise_012_split_brain_resolver_keep_oldest/src/main/resources/application.conf [hunk 8]
```diff
@@ -57,22 +57,4 @@ akka {
...
-    cluster-node-up-color          = green
-    cluster-node-weakly-up-color   = darkgreen
-    cluster-node-down-color        = red
-    cluster-node-left-color        = cyan
-    cluster-node-exited-color      = magenta
-    cluster-node-unreachable-color = whiteLow
-    cluster-node-joined-color      = orange
-  }
-}
+}
```
Comment (ironfish) [inline exercise_012_split_brain_resolver_keep_oldest/src/main/resources/application.conf]: Needs CRLF

File: exercise_013_split_brain_resolver_static_quorum_http_mamagement/src/main/resources/application.conf [hunk 9]
```diff
@@ -62,22 +62,4 @@ akka {
...
-    cluster-node-up-color          = green
-    cluster-node-weakly-up-color   = darkgreen
-    cluster-node-down-color        = red
-    cluster-node-left-color        = cyan
-    cluster-node-exited-color      = magenta
-    cluster-node-unreachable-color = whiteLow
-    cluster-node-joined-color      = orange
-  }
-}
+}
```
Comment (ironfish) [inline exercise_013_split_brain_resolver_static_quorum_http_mamagement/src/main/resources/application.conf]: Needs CRLF

**Comment by eloots** [comment]
Can you clarify please. Is it a missing CRLF at the end of the file or in other places? Are you referring to a certain style for config files?

Verstuurd vanaf mijn iPhone

**Pull request merged**